Service/MenuHydrator.php

```diff
@@ -17,9 +17,11 @@ class MenuHydrator
      */
     public function mapRequest(MenuInterface $menu, RequestInterface $request): MenuInterface
     {
-        $menu->setTitle($request->getParam('title'));
-        $menu->setIdentifier($request->getParam('identifier'));
-        $menu->setCssClass($request->getParam('css_class'));
+        $params = $request->getParams();
+        $menu->setTitle($params['title']);
```

Comment:
I think here and below lines we could use references from `MenuInterface` constants like: `MenuInterface::TITLE`

Reply:
Good idea, replaced with `MenuInterface` references.